stable/cosbench/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: cosbench
+version: 0.1.0
```

Comment:
set 1.0.0

Reply:
Sure I just update it.

Just wondering if this only applies to charts going to stable or is it a new rule (or one I missed)? Afaik, semantic versioning starts at `0.1.0`.

Reply:
When going to stable we start at 1.0.0